packages/cli/tests/unit/record/testCaseRecording.spec.ts

```diff
@@ -51,12 +55,24 @@ describe('record.testCaseRecording', () => {
 
       await TestCaseRecording.start(context);
       await TestCaseRecording.waitFor(context);
+      const fullExpectedCommand = process.platform === 'win32' ? 
+        'C:\\WINDOWS\\system32\\cmd.exe /d /s /c "node .\\tests\\sleeper.js 1000"' : 
+        '/bin/sh -c sleep 1';
+
+      // windows will throw an error when a process is killed using the pid
+      const windowsError = '\nTest command failed with status code 1: C:\\WINDOWS\\system32\\cmd.exe /d /s /c "node .\\tests\\sleeper.js 1000"';
```

Comment:
This constant could be moved down to the `if (process.platform)`; also it could use `fullExpectedCommand` to make it a little DRYer.

Reply:
Good idea!